x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
```

Comment:
Having a component with the same parent folder name is a bit confusing. Can you change it to `host_isolation_exceptions_page` or similar?

Reply:
I'll rename it to host_isolation_exceptions_list because that is what is showing :)